pypeit/core/fitting.py

```diff
@@ -629,21 +656,24 @@ def fit_gauss(x_out, y_out, guesses=None, w_out=None, nparam=3, maxfev=0):
 
     Args:
         x_out (`numpy.ndarray_`):
```

Comment:
`numpy.ndarray_` -->  `numpy.ndarray`_ 
for x_out, y_out, w_out

Reply:
edited